app/assets/javascripts/components/student_dashboard/pages/my_lessons_page.es6.jsx

```diff
@@ -43,17 +43,20 @@ class MyLessonsPage extends React.Component {
      <div className="page-wrapper">
       <Header />
       <div className="content-wrapper">
-        <div className="my-lessons-page content-container">
-          <h2 className="title">
-            My Lessons
-          </h2>
-          <div className="options-container">
-            <ButtonGroup>
-              {this.renderOption("upcoming")}
-              {this.renderOption("recent")}
-            </ButtonGroup>
-          </div>
-          <div className="lessons-container">
+        <div className="my-lessons-page">
+          <div className="content-container">
```

Comment:
same here, why don't you just put the styling on my-lessons-page instead of content-container